scripts/ci_checks/macos/standard-build.sh

```diff
@@ -15,7 +15,7 @@ scons -Q \
       --enable-examples \
       --enable-debug \
       --sanitizers=all \
-      --build-3rdparty=openfec \
+      --build-3rdparty=openfec,sndfile \
```

Comment:
On macos, libsndfile should be available via brew. So instead of adding sndfile to --build-3rdparty, we should add it to brew install command (above in this file).